Scoop/Model/User.swift

```diff
@@ -10,14 +10,15 @@ import UIKit
 
 class User {
```

Comment:
Should we still keep this User model? Are we going to be replacing it BaseUser completely?

Reply:
I'm not really sure yet. Since I'm just using the class rn to pass info through the VCs on the onboarding screen, since there is no backend implementation of the preferences. It's just a placeholder until I get more info on the Backend Route